src/foundry/client/core/workers.d.mts

```diff
@@ -122,19 +106,21 @@ declare global {
      */
     createWorker(name: string, config?: AsyncWorker.Options): Promise<AsyncWorker>;
 
-    /**
-     * Get a currently active Worker by name.
-     * @param name - The named Worker to retrieve
-     * @returns The AsyncWorker instance
-     */
-    getWorker(name: string): AsyncWorker;
-
     /**
      * Retire a current Worker, terminating it immediately.
      * @see {@link Worker#terminate}
      * @param name - The named worker to terminate
      */
     retireWorker(name: string): void;
+
+    /**
+     * Get a currently active Worker by name.
+     * @param name - The named Worker to retrieve
+     * @returns The AsyncWorker instance
+     * @deprecated since v11, will be removed in v13
+     * @remarks `"WorkerManager#getWorker is deprecated in favor of WorkerManager#get"`
+     */
+    getWorker(name: string): AsyncWorker;
```

Comment:
I don't know if it's worth mentioning somewhere that this method throws an error if name is not in the internal map, where `get` doesn't.